uni/lib/controller/parsers/parser_library_occupation.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:uni/model/entities/library_occupation.dart';
+
+Future<LibraryOccupation> parseLibraryOccupationFromSheets(
+    Response response) async {
+  final js = response.body.split('\n')[1];
+  const skip = 'google.visualization.Query.setResponse(';
```

Comment:
This is very cryptic. Can you improve the variable naming, or perhaps add a comment?